packages/orchestrator-ui-components/src/components/WfoTable/WfoTable/styles.ts

```diff
@@ -136,14 +136,31 @@ export const getWfoTableStyles = ({ theme, isDarkThemeActive }: WfoTheme) => {
         height: '100%',
         zIndex: theme.levels.modal,
         '&:hover, &:active': {
+            transition: 'background-color 0.15s',
             backgroundColor: isDarkThemeActive
                 ? theme.colors.mediumShade
                 : theme.colors.header,
         },
         '&::after': {
-            content: '""',
+            display: 'flex',
+            paddingTop: theme.base - 2,
```

Comment:
`base - 2` is tricky, because that does not scale. Try to find a nice value to devide `/` instead of minus `-`